ngViewer/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html ng-app="ngViewer">
+<head>
+
+    <meta charset="utf-8">
+
+    <!--CSS-->
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css"/>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap-theme.min.css"/>
+    <link rel="stylesheet" type="text/css" href="font-awesome/css/font-awesome.min.css"/>
```

Comment:
what are the different `font-awesome/*` files for if `font-awesome/css/font-awesome.min.css` is the only one used.